test/server/drain_manager_impl_test.cc

```diff
@@ -37,12 +40,15 @@ TEST(DrainManagerImplTest, All) {
   // Test drain sequence.
   Event::MockTimer* drain_timer = new Event::MockTimer(&server.dispatcher_);
```

Comment:
@alyssawilk who had a question on the Envoy-ish pattern for this form of memory management the other day.